python/common/python/math.java

```diff
@@ -0,0 +1,23 @@
+package python;
+
+@org.python.Module()
+public class math extends org.python.types.Module {
+
+    @org.python.Method(
+        __doc__ = "Return the sine of x radians.\n",
+        args = {"x"})
+    public static org.python.types.Float sin(org.python.Object x)  {
+        // Bool, Ints, Floats allowed
+        if (x instanceof org.python.types.Int){
+            return new org.python.types.Float(Math.sin(Double.valueOf(((org.python.types.Int) x.__int__()).value)));
+        }
+        if (x instanceof org.python.types.Float){
+            return new org.python.types.Float(Math.sin(Double.valueOf(((org.python.types.Float) x.__float__()).value)));
+        }
+        if (x instanceof org.python.types.Bool){
+            return new org.python.types.Float(Math.sin(Double.valueOf(((org.python.types.Float) x.__float__()).value)));
+        }            
```

Comment:
Agree with @brechtcallens about returning with proper vairable names might be good for readability! Doing this might also make it possible to remove the comment on line [10](https://github.com/brechtcallens/voc/pull/3/files#diff-77984473c993dac1be449185449dd0afc7a770fa94b880e308c9e4581c975086R10) :building_construction:

Reply:
Solved

Reply:
Solved in 4f16cbc883993ef5eacab7a492bd5774135e29e8